os/linux/gbmbufferhandler.cpp

```diff
@@ -85,9 +85,9 @@ bool GbmBufferHandler::CreateBuffer(uint32_t w, uint32_t h, int format,
   temp->import_data.height = gbm_bo_get_height(bo);
   temp->import_data.format = gbm_bo_get_format(bo);
 #if USE_MINIGBM
-  temp->import_data.fds[0] = gbm_bo_get_plane_fd(bo, 0);
   size_t total_planes = gbm_bo_get_num_planes(bo);
   for (size_t i = 0; i < total_planes; i++) {
+    temp->import_data.fds[i] = gbm_bo_get_plane_fd(bo, i);
```

Comment:
@js0701 Can you check what needs to be done here too:
https://github.com/intel/IA-Hardware-Composer/blob/master/os/android/utils_android.h#L165